react-search-component/my-search-component/src/Components/SearchComponent.jsx

```diff
@@ -0,0 +1,27 @@
+import SearchBar from './SearchBar';
+import ListItems from './ListItems';
+import { useState } from 'react';
+const list = [
```

Comment:
I would prefer to see this declared in App and then passed to SearchComponent as a prop. That would make it more similar to how it will work when we introduce the backend.